docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -83,7 +93,77 @@ export default function App() {
 
 ## Configuration
 
-To configure `expo-splash-screen`, see the following [app config](/workflow/configuration/) properties.
+You can configure `expo-splash-screen` using its built-in [config plugin](/config-plugins/introduction/) if you use config plugins in your project ([EAS Build](/build/introduction) or `npx expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect. If your app does **not** use EAS Build, then you'll need to manually configure the package.
+This is now the preferred method of configuring the splash screen. The other methods are now considered legacy and will be removed in the future.
+
+<ConfigPluginExample>
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          "backgroundColor": #232323,
+          "image": "./assets/splash-icon.png",
+          "dark": {
+            "image": "./assets/splash-icon-dark.png",
+            "backgroundColor": "#000000"
+          },
+          "imageWidth": 200
+        }
+      ]
+    ],
+  }
+}
+```
+</ConfigPluginExample>
+
+<ConfigPluginProperties
+  properties={[
+    {
+      name: 'backgroundColor',
+      description: 'A hex color string representing the background color of the splash screen.',
+      default: '#ffffff',
+    },
+    {
+      name: 'image',
+      description:
+        'The path to the image file that will be displayed on the splash screen. This should be your app icon or logo.',
+      default: 'undefined',
+    },
+    {
+      name: 'enableFullScreenImage_legacy',
+      description:
+        'Enabling this property allows using a full screen image as the splashscreen. This is to help with the transition from the legacy splash screen configuration and will be removed in the future.',
+      platform: 'ios',
+      default: 'false',
+    },
+    {
+      name: 'dark',
+      description:
+        'An object containing properties for configuring the splash screen when the device is in dark mode.',
+      default: 'undefined',
+    },
+    {
+      name: 'imageWidth',
+      description: 'The width to make the image.',
+      default: '100',
```

Comment:
```suggestion
      default: '200',
```

After creating a new project with the default template, I see the default value is `200`. Let's update this.

![CleanShot 2024-11-08 at 00 59 06](https://github.com/user-attachments/assets/6d69291d-b1c7-4b42-a6b8-97dfa1482b04)

Reply:
Brent added this to the template to try and keep some backward compatibility with the old config but if you don't provide a value for `imageWidth` the default internally is 100

Reply:
Thanks for the clarification!